src/AmalgamatorSession.ts

```diff
@@ -675,6 +696,37 @@ export class AmalgamatorSession extends LoggingDebugSession {
             ].customRequest('cdt-gdb-adapter/Memory', args);
             response.body = childResponse.body;
             this.sendResponse(response);
+        } else if (command === 'cdt-amalgamator/resumeAll') {
+            const [, threads] = await this.collectChildTheads();
+            for (const thread of threads) {
+                const [childIndex, childId] = await this.getThreadInfo(
+                    thread.id
+                );
+                const childDap = this.childDaps[childIndex];
+                const childResponse = await childDap.threadsRequest();
+                for (const threadOfChildDap of childResponse.body.threads) {
+                    const ThreadInfo = threadOfChildDap as ThreadInfo;
+                    if (ThreadInfo.running === false) {
```

Comment:
Please use lower case to start variable names.

```suggestion
                    const threadInfo = threadOfChildDap as ThreadInfo;
                    if (threadInfo.running === false) {
```